tools/topology/topology2/cavs-nocodec.conf

```diff
@@ -136,11 +139,12 @@ Object.Dai {
 #
 # Pipeline definitions
 #
-# PCM0 ---> gain ----> Mixin ----> Mixout ----> gain ----> SSP0
+# PCM0 ---> gain ----> Mixin ----> Mixout ----> gain ----> smart_amp ----> SSP0
```

Comment:
I'm guessing this would be the dummy smart amp for CI tests ?

Reply:
right this is with the smart_amp_test module